docusaurus.config.js

```diff
@@ -179,9 +180,9 @@ const config = {
             to: "/api/subgraphs/overview",
           },
           {
-            label: "Discord",
+            label: "Links",
             position: "right",
-            to: "https://discord.gg/bSwRCwWRsT",
+            to: "https://linktr.ee/SablierLabs",
```

Comment:
Happy to add this Links button in the navbar but I would also keep Discord to increase the conversion rate.

Reply:
Will do.